tests/integration/targets/ec2_snapshot/tasks/main.yml

```diff
@@ -264,10 +286,10 @@
         max_results: 5
       register: info_result
 
-    - assert:
-        that:
-          - info_result.snapshots | length == 5
-          - info_result.next_token_id is defined
+#    - assert:
```

Comment:
Can we correct the assert statements or is there a specific reason behind commenting on?